### ISSUE TITLE
Error in heuristics.py

### ISSUE THREAD

**Issue by SimDeh** [FRESNA/powerplantmatching#5]
Hey guys,

I spotted an issue caused by code in "heuristics.py" and "collection.py"

```
File "powerplantmatching/collection.py", line 117, in MATCHED_dataset
    matched = extend_by_non_matched(matched, OPSD(), 'OPSD', clean_added_data=True)

  File "powerplantmatching/heuristics.py", line 52, in extend_by_non_matched
    not_included = pd.DataFrame(extend_by.projectID.tolist())

  File "anaconda/envs/py35/lib/python3.5/site-packages/pandas/core/generic.py", line 2744, in __getattr__
    return object.__getattribute__(self, name)

AttributeError: 'DataFrame' object has no attribute 'projectID'
```

Best regards

**Comment by coroa** [comment]
That has already been fixed in commit 5b50146. Please pull the latest changes and confirm!

**Comment by SimDeh** [comment]
I've pulled the latest changes and as it seems it works!
